src/dev/i18n/extractors/html.test.js

```diff
@@ -80,7 +80,7 @@ describe('dev/i18n/extractors/html', () => {
     expect(() => extractHtmlMessages(source).next()).toThrowErrorMatchingSnapshot();
   });
 
-  test('throws on i18n filter usage in angular directive argument', () => {
+  test('throws on i18n filter usage in complex angular expression', () => {
```

Comment:
__question:__ Do we have test for something like this?
```html
 content="::'regionMap.visParams.switchWarningsTipText' | i18n: {
            defaultMessage: 'Turns on/off warnings. When turned on, warning will be shown for each term that cannot be matched to a shape in the vector layer based on the join field. When turned off, these warnings will be turned off.'
          }"
```

If no, could you please add one?